src/context/steps.ts

```diff
@@ -14,6 +15,9 @@ export abstract class BaseLazyStep<TResult = unknown> {
   public readonly stepName;
   public abstract readonly stepType: StepType; // will be set in the subclasses
   constructor(stepName: string) {
+    if (!stepName) {
+      throw new WorkflowError("step name can't be undefined or empty string.");
```

Comment:
A workflow step name cannot be undefined or an empty string. Please provide a name for your workflow step.